manual/python/index.md

```diff
@@ -597,3 +597,26 @@ modifier methods on the `std::string` objects.
 <class cppyy.gbl.std.string at 0x4cd41b0>
 <class cppyy.gbl.std.string at 0x4cd41b0>
 ```
+
+- When obtaining the boolean value of a C++ instance proxy, both old and new PyROOT return
+`False` when such proxy points to null. On the other hand, when the proxy points to a C++ object,
```

Comment:
```suggestion
`False` when such proxy points to null. On the other hand, when the proxy points to a valid C++ object,
```

Reply:
Indeed, I thought about this, but then I realized that even if the object is not valid (e.g. it has been already deleted without the proxy knowing it), it could still return `True`.